.travis.yml

```diff
@@ -43,6 +43,6 @@ script:
   - ./etc/bin/parallel_phpunit.sh
 
 notifications:
-  slack: bulutyazilim:8GtvUiq3935dZRc1SObHJVcR#ojs-log
+  slack: ideafield:8GtvUiq3935dZRc1SObHJVcR#ojs-log
   email:
```

Comment:
@yazbahar can you remove my email address too. i will follow from slack channel.